### PR TITLE
finished the feature of many to many

### PR DESCRIPTION
**Pull request by aeo03**

### CODE REVIEW

File: app/views/quotations/show.html.erb [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 
 <p><strong>Quote: </strong><%= @quotation.quote %></p>
-<p><strong>Author: </strong><%= @quotation.creator.author if @quotation.creator %></br></br>
+<p><strong>Author: </strong><%= @quotation.creator.author if @quotation.creator %>
+<p><strong>Tags: </strong></br>
+
+<tbody>
+        <% @quotation.tags.each do |tag| %>
+            <tr>
+                <td><%= tag.tag_name if @quotation.tags %></td>
```
Comment (analyn-cajocson) [inline app/views/quotations/show.html.erb]: remove `if @quotation.tags` inside the `each loop`. You can put it before `each loop`

Comment (aeo03) [reply app/views/quotations/show.html.erb]: Noted

File: test/test_helper.rb [hunk 2]
```diff
@@ -11,7 +11,11 @@ class ActiveSupport::TestCase
...
-  def sign_in_as(user)
-    post auth_user_path, params: { username: user.username, password: "password123" }
+    def sign_in_as(user)
+      post auth_user_path, params: { username: user.username, password: "password123" }
+    end
+
   end
-end
+
+
```
Comment (analyn-cajocson) [inline test/test_helper.rb]: remove unnecessary white space

Comment (aeo03) [reply test/test_helper.rb]: Noted

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**